index.bs

```diff
@@ -1322,15 +1322,38 @@ Verifiable Random Selection Procedure</h5>
 			The incomplete terms are assigned in result order.
 	</ol>
 
+<h5 id="AB-TAG-removal">
+Removing AB or TAG members</h5>
+
+	Individual participants of the [=Advisory Board=] or the [=Technical Architecture Group=]
+	can be <dfn export lt="remove|removal">removed</dfn> from those groups
+	if they are found by their peers
+	to be grossly neglecting their duties,
+	or to be acting in a way that seriously hampers the group's ability to function normally.
+
+	A chair of the [=AB=] or [=TAG=] <em class=rfc2119>must</em> hold a hearing
```

Comment:
There should be explicit detail here about who is invited to that hearing; both who must be invited (I'd expect the relevant body, including the individual) and who is allowed.  Also, whether this is publicized or not.  (I would expect not, but it shouild be explicit that it's confidential, perhaps?

Reply:
created https://github.com/w3c/process/issues/1007 to follow up on that